index.js

```diff
@@ -86,6 +87,7 @@ app.use(
         ...createLoaders(accessToken, userID, requestID),
       },
       formatError: graphqlErrorHandler(request.body),
+      validationRules: [depthLimit(10, { ignore: [] }, depths => console.log("Depths: ", depths))],
```

Comment:
This produced a JSDoc lint error, fixed here: https://github.com/stems/graphql-depth-limit/pull/1